drivers/gles3/shaders/scene.glsl

```diff
@@ -1270,17 +1270,62 @@ LIGHT_SHADER_CODE
 }
 
 float sample_shadow(highp sampler2DShadow shadow, vec2 shadow_pixel_size, vec2 pos, float depth, vec4 clamp_rect) {
-#ifdef SHADOW_MODE_PCF_13 //ubershader-runtime
+	float avg = textureProj(shadow, vec4(pos, depth, 1.0));
+#ifdef SHADOW_MODE_PCF_LOW //ubershader-runtime
+#ifdef SHADOW_MODE_PCF_HIGH //ubershader-runtime
```

Comment:
Can we do this? :thinking: 
```
#if defined(SHADOW_MODE_PCF_LOW) && defined(SHADOW_MODE_PCF_LOW)
```
This pattern seems to be used elsewhere in the shaders and it seems easier to follow.

Reply:
I think this messes with `//ubershader-runtime`, unfortunately. We can use this approach in 4.x though where ubershaders aren't used in Compatibility.